### PR TITLE
Spanish Translation Miquel Vila Porte

### PR DESCRIPTION
**Pull request by alpita-masurkar**
## Description

PR for Spanish language support

### CODE REVIEW

File: app/locales/languages.js [hunk 1]
```diff
@@ -57,7 +58,8 @@ i18next.init({
   resources: {
     en: { label: 'English', translation: en },
     ht: { label: 'Kreyòl ayisyen', translation: ht },
-    it: { label: 'Italian', translation: it },
+    it: { label: 'Italiano', translation: it },
+    es: { label: 'Español', translation: es },
```
Comment (tstirrat) [inline app/locales/languages.js:62]: sort this above ht

File: i18next-parser.config.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ module.exports = {
   lineEnding: 'auto',
   // Control the line ending. See options at https://github.com/ryanve/eol
 
-  locales: ['en', 'ht', 'it'],
+  locales: ['en', 'ht', 'it', 'es'],
```
Comment (tstirrat) [inline i18next-parser.config.js:41]: sort these alphabetically

File: app/locales/languages.js (hunk 1 above)
Comment (alpita-masurkar) [reply app/locales/languages.js]: done

File: i18next-parser.config.js (hunk 2 above)
Comment (alpita-masurkar) [reply i18next-parser.config.js]: done

**Comment by alpita-masurkar** [comment]
alphabetically sorted. thank you for reviewing it again :)

**Pull request merged**